assets/js/components/settings/SettingsRenderer.js

```diff
@@ -75,7 +75,9 @@ export default function SettingsRenderer( { slug } ) {
 
 	if ( isEditing && SettingsEditComponent ) {
 		return <SettingsEditComponent />;
-	} else if ( ! isEditing && SettingsViewComponent ) {
+	} else if ( isEditing && ! SettingsEditComponent ) {
+		return <SettingsViewComponent />;
+	} else if ( SettingsViewComponent ) {
 		return <SettingsViewComponent />;
```

Comment:
@aaemnnosttv This can be simplified to just a single `else if ( SettingsViewComponent )` check. Right now it might result in an error if `SettingsViewComponent` is not set.

Reply:
I remember thinking about this when I changed it initially and then changed it back, but I don't remember why 😄  Updated!